### PR TITLE
[fix] Delete label studio dependency in label-studio-ml

### PR DESCRIPTION
**Pull request by KonstantinKorotaev**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,10 @@
...
+attrs>=19.2.0
+appdirs>=1.4.3
+colorama>=0.4.4
 Flask==1.1.2
-label-studio>=1.0.0
+lxml>=4.2.5
+Pillow
+requests>=2.22.0,<3
+scikit-learn==0.24.1
+label-studio-tools==0.0.0.dev8
```
Comment (niklub) [inline requirements.txt:10]: should we update here to the latest?

File: label_studio_ml/model.py [hunk 2]
```diff
@@ -667,3 +671,26 @@ def webhook(cls, event, data):
         job_id = cls._generate_version()
         cls.get_job_manager().run_job(cls.model_class, (event, data, job_id))
         return {'job_id': job_id}
+
+
+def get_all_classes_inherited_LabelStudioMLBase(script_file):
```
Comment (niklub) [inline label_studio_ml/model.py:676]: why this one has been moved?

File: label_studio_ml/model.py [hunk 3]
```diff
@@ -16,12 +17,15 @@
 from rq import Queue, get_current_job
 from rq.registry import StartedJobRegistry, FinishedJobRegistry, FailedJobRegistry
 from rq.job import Job
+from colorama import Fore
+import importlib
```
Comment (niklub) [inline label_studio_ml/model.py:21]: order

File: label_studio_ml/utils.py [hunk 4]
```diff
@@ -1,51 +1,17 @@
...
-            names.add(name)
-        for base in obj.__bases__:
-            if LabelStudioMLBase.__name__ == base.__name__:
-                names.add(name)
-    sys.path.pop()
-    names = list(names)
-    return names
+_LABEL_TAGS = {'Label', 'Choice'}
+_NOT_CONTROL_TAGS = {'Filter',}
+_DIR_APP_NAME = 'label-studio'
```
Comment (niklub) [inline label_studio_ml/utils.py:14]: where this one is used?

**Review by niklub — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 1 above)
Comment (KonstantinKorotaev) [reply requirements.txt]: Fixed

File: label_studio_ml/utils.py (hunk 4 above)
Comment (KonstantinKorotaev) [reply label_studio_ml/utils.py]: Obsolete, deleted

File: label_studio_ml/model.py (hunk 2 above)
Comment (KonstantinKorotaev) [reply label_studio_ml/model.py:676]: Refactored cycle dependency

**Review by niklub — APPROVED** [review]

**Pull request merged**